core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -912,8 +912,22 @@ ExWorkProcRetcode ExHbaseAccessUpsertVsbbSQTcb::work()
 	    copyRowIDToDirectBuffer(rowId_);
 
 	    currRowNum_++;
-	    matches_++;
-
+            // temp code below.WIll remove the envvar after this
+            //has got enough exposure in  testing.
+            if (getenv("TURN_OFF_RETURNROW")) 
```

Comment:
Thanks - I will remove this. I had used it for quick debugging. I didn't realize it'll have an impact on performance.  Will look into fixing it and submit a new version of this file.